server.js

```diff
@@ -70,6 +72,25 @@ async function renderAndCache(req, res, pagePath, serverData) {
       appMiningApps,
       params: req.params
     }
+
+    if (req.universalCookies.get('jwt')) {
```

Comment:
Some downsides here - this will run on every page if the user is logged in. It'll need to be tested, but you could probably refactor this to only run if `req.path.indexOf('maker') !== -1`